src/Model/Svg/SvgFile.php

```diff
@@ -202,8 +203,8 @@ protected function makeTranslationReady(): bool
             if ($tspan->childNodes->length > 1
                 || ( 1 == $tspan->childNodes->length && XML_TEXT_NODE !== $tspan->childNodes->item(0)->nodeType )
             ) {
-                $this->logFileProblem('File {file} has nested <tspan> tags');
-                return false; // Nested tspans not (yet) supported
+                // Nested tspans not (yet) supported. T250607.
+                throw new NestedTspanException($tspan);
```

Comment:
(Non-blocking question) I know that you are throwing a new Exception, but maybe we can also log the message thrown, or is this log message not being used anyway?

Reply:
Yeah, I wondered about that too. I went with the same pattern that's already used for SvgLoadException, which is also caught by the controller and used to display a message to the user.

I guess the reason for not logging is that this is not an unexpected condition, and so we don't want to read about it in the logs. (Of course, we don't actually look at the logs much these days! :rofl:)